app/helpers/duplicate_logs_helper.rb

```diff
@@ -0,0 +1,37 @@
+module DuplicateLogsHelper
+  def duplicates_for_user(user)
+    duplicate_sets = { lettings: {}, sales: {} }
+    lettings_count = 0
+    sales_count = 0
+    duplicate_lettings_ids = Set.new
+    duplicate_sales_ids = Set.new
+
+    user.lettings_logs(created_by: true).each do |log|
+      next if duplicate_lettings_ids.include? log.id
+
+      duplicates = LettingsLog.filter_by_organisation(user.organisation).duplicate_logs(log)
```

Comment:
`user.lettings_log` would return all the logs available for this user, might be safer if we ever change the scope of what's available. Also might be worth adding `.visible` to avoid listing all the deleted logs

Reply:
have changed the first bit. duplicate logs scope has visible in it so I won't repeat that.

Reply:
Hm, I think it was showing me deleted logs as well. Does it only check if the duplicates are visible, or the log that gets passed is as well?